src/Common/src/System/SR.cs

```diff
@@ -32,6 +32,11 @@ private static bool UsingResourceKeys()
             return false;
         }
 
+        internal static string GetResourceString(string resourceKey)
+        {
+            return GetResourceString(resourceKey, resourceKey);
```

Comment:
Is this used anywhere since the code was changed to use strongly-typed resources?